src/query_parser/v2/parser.y

```diff
@@ -641,74 +594,72 @@ termlist(A) ::= param_term(B) param_term(C). [TERMLIST]  {
 }
 
 termlist(A) ::= termlist(B) param_term(C) . [TERMLIST] {
-    A = B;
-    if (!(C.type == QT_TERM && StopWordList_Contains(ctx->opts->stopwords, C.s, C.len))) {
```

Comment:
Where did the stop-word check go?

Reply:
This might be an oversigt, but we only check this here (`termlist(A) ::= termlist(B) param_term(C)`) but not in the initial rule of the term list (`termlist(A) ::= param_term(B) param_term(C)`).
Do we need to filter stop-words in a tag search? If so, we should check it for the first two terms as well. I was under the assumption we don't need to filter stop-words for tag searches

Reply:
OK, this is a bug. We should validate that the term is not a stop word, and we miss this filtering in the base rule.
I'll revert this change, and we should fix the bug in another PR.